kratos/includes/kratos_filesystem.h

```diff
@@ -21,7 +21,7 @@
 
 // Project includes
 #include "includes/define.h"
-
+#include "includes/model_part.h"
```

Comment:
Maybe you can do a forward declaration to avoid the include, opinion @philbucher ?